.github/workflows/integration_test.yaml

```diff
@@ -300,6 +321,26 @@ jobs:
           EXPERIMENT_NAME: ${{ inputs.chaos-experiments }}
           TOTAL_CHAOS_DURATION: ${{ inputs.chaos-duration }}
           KUBE_CONFIG_DATA: ${{ env.KUBE_CONFIG_DATA }}
+      - name: Install k6s
+        if: ${{ inputs.load-test-enabled }}
+        run: sudo snap install k6
+      - name: Run k6 load tests
+        if: ${{ inputs.load-test-enabled }}
+        run: k6 run load_tests/load-test.js ${{ inputs.load-test-run-args }}
+      - name: Run Github Trivy FS Action
```

Comment:
Any reason why trivy is run before k6s (when the last one runs)?

Reply:
I changed k6 to run before ZAP/Trivy scanning actually because ZAP full scan was simulating attacks and impacting Indico performance. So when k6 ran after the scanning, the http_req_duration was not as expected.
Example:
https://github.com/canonical/indico-operator/actions/runs/3952422613/jobs/6767561531

I changed for the baseline scan that is faster but even so left k6 first.

Reply:
Thanks for clarifying